diff --git a/CMakeLists.txt b/CMakeLists.txt
index 50a80629..557f9cc2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,7 +100,7 @@ if(CMAKE_HIP_COMPILER)

     if(NOT AMDGPU_TARGETS)
         find_package(hip REQUIRED)
-        list(FILTER AMDGPU_TARGETS INCLUDE REGEX "^gfx(94[012]|101[02]|1030|110[012]|120[01])$")
+        list(FILTER AMDGPU_TARGETS INCLUDE REGEX "^gfx(94[012]|101[02]|1030|110[012]|115[01]|120[01])$")
     endif()

     if(WIN32 AND WINDOWS_AMDGPU_TARGETS_EXCLUDE_REGEX)
diff --git a/CMakePresets.json b/CMakePresets.json
index 72417ade..19185f7e 100644
--- a/CMakePresets.json
+++ b/CMakePresets.json
@@ -68,7 +68,7 @@
       "inherits": [ "ROCm" ],
       "cacheVariables": {
         "CMAKE_HIP_FLAGS": "-parallel-jobs=4",
-        "AMDGPU_TARGETS": "gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1151;gfx1200;gfx1201;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-"
+        "AMDGPU_TARGETS": "gfx1150"
       }
     },
     {
